src/main/java/com/softserve/demo/controller/OfferController.java

```diff
@@ -1,55 +1,50 @@
 package com.softserve.demo.controller;
 
-import com.softserve.demo.model.Offer;
+import com.softserve.demo.dto.OfferDTO;
 import com.softserve.demo.service.OfferService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.data.domain.Page;
+import com.softserve.demo.util.OfferMapper;
 import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.*;
 
+import javax.validation.Valid;
+import java.util.List;
+
 @CrossOrigin("*")
 @RestController
 @RequestMapping("offers")
 public class OfferController {
 
-    @Autowired
-    OfferService offerService;
+    private final OfferService offerService;
+    private final OfferMapper offerMapper;
 
-    @PostMapping
-    public ResponseEntity<?> createOffer(@RequestBody Offer offer) {
-        offerService.createOffer(offer);
-        return new ResponseEntity<>(HttpStatus.CREATED);
-    }
+    public OfferController(OfferService offerService, OfferMapper offerMapper) {
 
```

Comment:
Do you want add smth here?

Reply:
Sorry, but what do you mean?

Reply:
21 row is empty and it is inside constructor

Reply:
OK